### PR TITLE
feat(forwarder): add edgehog-forwarder member

### LINKED ISSUE #273: Add forwarder member to the device-runtime workspace

### PR DESCRIPTION
**Pull request by rgallor**
Add edgehog-forwarder as a workspace package.

Closes #273

### CODE REVIEW

File: .reuse/dep5 [hunk 1]
```diff
@@ -5,3 +5,7 @@ Source: https://github.com/edgehog-device-manager/edgehog-device-runtime
 Files: CHANGELOG.md
 Copyright: 2022 SECO Mind Srl
 License: CC0-1.0
+
+Files: edgehog-forwarder/interfaces/*
+Copyright: 2022 SECO Mind Srl
+License: CC0-1.0
```
Comment (harlem88) [inline .reuse/dep5:11]: Remove these lines, the interfaces should be added  into the [edgehog-astarte-interfaces repo](https://github.com/edgehog-device-manager/edgehog-astarte-interfaces).
```suggestion
```

**Review by harlem88 — CHANGES_REQUESTED** [review]

File: Cargo.toml [hunk 2]
```diff
@@ -13,9 +13,10 @@ rust-version = { workspace = true }
 [workspace]
 resolver = "2"
 members = [
-  "e2e-test",
-  "hardware-id-service",
-  "led-manager-service",
+    "e2e-test",
+    "edgehog-forwarder",
```
[the remark this answers is not in the document]
Comment (rgallor) [reply Cargo.toml]: Done

File: .reuse/dep5 (hunk 1 above)
Comment (rgallor) [reply .reuse/dep5]: Done

**Review by joshuachp — APPROVED** [review]
LGTM

**Review by harlem88 — APPROVED** [review]

**Pull request merged**